components/emoji_picker/emoji_picker.jsx

```diff
@@ -486,7 +482,7 @@ export default class EmojiPicker extends React.PureComponent {
         }
     }
 
-    onSkinSelected = (skin) => {
+    handleSkinChange = (skin) => {
         this.props.actions.setUserSkinTone(skin);
```

Comment:
Probably this can be moved to the the other component using a useDispatch, but if not, we can at least pass directly the `this.props.actions.setUserSkinTone`.